### ISSUE TITLE
False warning on gems for :ruby platform in Gemfile

### ISSUE THREAD

**Issue by yltsrc** [jruby/jruby-lint#15]
I have Gemfile with content

```
source 'https://rubygems.org'
gem 'mysql2', platforms: :ruby
gem 'activerecord-jdbcmysql-adapter', platforms: :jruby
gem 'therubyracer', platforms: :ruby
gem 'therubyrhino', platforms: :jruby
```

and I get false warnings:

```
./Gemfile:5: [gems, warning] Found gem 'mysql2' which is reported to have some issues:
Use activerecord-jdbcmysql-adapter instead.
./Gemfile:23: [gems, warning] Found gem 'therubyracer' which is reported to have some issues:
Try using therubyrhino instead.
```

**Comment by yltsrc** [comment]
From my point of view the best way would be to show warning only if right gem is missing in Gemfile or not existing for jruby platform
